### PR TITLE
Testing - Part 1

### PR DESCRIPTION
**Pull request by cmolinan**
### UPDATES FOR TESTING REQUIREMENTS
- Using jest framework for testing, create a test file containing the add item and delete item functions that you want to test.
- Mock HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM.
- Make sure you group your tests using the describe() method.

### CODE REVIEW

File: __tests__/library.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+});
+
+test('Removing a task', () => {
+  eraseToDoList();
+  addToDo('Save the Queen');
+  addToDo('Jump to the sky');
+  addToDo('Go to the beach');
+  removeList(1);
+  expect(toDoList).toHaveLength(2);
+});
```
Comment (mikethreels) [inline __tests__/library.test.js:42]: - [ ] test should be grouped together using the `describe` block, however this test is outside of any block, please make sure to adjust this before requesting another review.

Comment (mikethreels) [inline __tests__/library.test.js]: - [x] test should be grouped together using the `describe` block, however this test is outside of any block, please make sure to adjust this before requesting another review.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: __tests__/library.test.js (hunk 1 above)
Comment (DhuruvSingh) [reply __tests__/library.test.js]: The requested changes have been done 👍

**Review by topeogunleye — APPROVED** [review]
### Hi Team, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**